Firestore/core/src/util/ordered_code.h

```diff
@@ -122,6 +151,21 @@ class OrderedCode {
   OrderedCode& operator=(const OrderedCode&) = delete;
```

Comment:
Could you please share with me why we don't need to delete move ctor and move assignment operator here?

Reply:
My little research seems to suggest they should be deleted explicitly as well, at least there would be no harm. Thanks.